cmd/minishift/cmd/start.go

```diff
@@ -281,11 +295,14 @@ func startHost(libMachineClient *libmachine.Client) (*host.Host, string) {
 		atexit.ExitWithMessage(1, fmt.Sprintf("Error starting the VM: %v", err))
 	}
 
+	print("-- Checking for IP ... ")
```

Comment:
Already concluded this when looking at the code here: https://github.com/gbraad/minishift/commit/802ea86e070e194d49deef88bc863e48f80f427d. It seems the original code came TOO late to determine an IP was not assigned. Will correct this...